### ISSUE TITLE
ex USSR countries

### ISSUE THREAD

**Issue by chaotism** [novafloss/workalendar#196]
Could you add ex USSR countries to calendar? Or better way is add countries in pull request?

**Comment by Natim** [comment]
Yes if you could start a pull request and especially tests for the last year and the next year it would be a really good start for us to help you add this.

**Comment by brunobord** [comment]
@chaotism I agree with @Natim - I think you could make a pull-request to include at least one country and see how it goes, using the pull-request template [and the documentation](https://github.com/novafloss/workalendar/blob/master/CONTRIBUTING.rst#adding-new-calendars) that should be easy.
